modules/cake.js

```diff
@@ -160,35 +159,17 @@ Module.addEvent("ready", () => {
     u.errorHandler(e, "Cakeday Init");
   }
 })
-// Janky stuff, but it works!!! (for now lol)
-.setUnload((date, type) => {
-  if (type === "cake") testCakeDays(undefined, date);
-  else if (type === "bday") testBirthdays(undefined, date);
-})
-.addCommand({ name: "bday",
```

Comment:
keep the development commands. they're there for a reason

Reply:
I didn't understand that they did in fact still work. I was probably trying them wrong. brought them back.